src/pages/home/Home.tsx

```
@@ -1,54 +1,47 @@
-import { Icon } from '@shared/components/icon/Icon';
+import { useState } from 'react';
+
+import BottomSheet from '@shared/components/bottom-sheet/BottomSheet';
+import ButtonDate from '@shared/components/button-date/ButtonDate';
+import Calendar from '@shared/components/calendar/Calendar';
+import { type SelectedDate } from '@shared/components/calendar/Calendar';
```

ESLint

[error] '@shared/components/calendar/Calendar' import is duplicated.

(no-duplicate-imports)